src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -1131,11 +1131,18 @@ paths:
       tags:
         - snapshots
         - repository
-      description: Add an auth domain to the snapshot. *** WARNING - Once this is set it cannot be modified ***
-                  A Sam auth domain consists of a collection of user groups. If a snapshot has an auth domain,
-                  users must have access to the resource directly (via a policy) and belong to all user groups
-                  in the auth domain in order to view and export the data. For example, if a user is a snapshot
-                  steward but is not included in the auth domain, Sam will prevent that user from accessing the resource.
+      description: >
+        Add data access control groups to the snapshot.  The provided user groups will be registered
+        as a group constraint policy in Terra Policy Service, and as an auth domain in Sam.
+
+
```

Comment:
FYI -- double new lines are necessary to render a line break in Swagger, which can help readability.